module-gui/gui/SwitchData.hpp

```diff
@@ -17,14 +17,22 @@ namespace gui
     /// if requested)
     class SwitchData
     {
+        bool popup = false;
```

Comment:
hmmm why ? we should have that info in Disposition or WindowsData ?

Reply:
aand it was fixed :)